ui-v2/app/controllers/dc/nodes/show.js

```diff
@@ -15,7 +15,7 @@ export default Controller.extend(WithFiltering, {
   },
   setProperties: function() {
     this._super(...arguments);
-    set(this, 'selectedTab', 'health-checks');
+    set(this, 'selectedTab', get(this.item, 'Checks.length') > 0 ? 'health-checks' : 'services');
```

Comment:
Good point, added in some comments here. From what I understand this method literally called just after I do my setProperties in the route. The big thinking for doing it here and not in the route, is because this variable is only there for frontend/view layer reasons, therefore it should live/be set on the Controller/ViewController. If the View/design should change, then the Route shouldn't be changed, the Controller should.